updates/0.0.1-admins.js

```diff
@@ -9,7 +9,7 @@
 
 exports.create = {
 	User: [
-		{ 'name.first': 'Admin', 'name.last': 'User', 'email': 'omisi2018@gmail.com', 'password': 'admin', 'isAdmin': true },
+		{ 'name.first': 'Admin', 'name.last': 'User', 'email': 'adminuser@shecodeafrica.com', 'password': 'anadmin', 'isAdmin': true },
```

Comment:
Is there a way to abstract these admin details without breaking the app? I don't think its best having them exposed

Reply:
I will look into this. The details are dummy data but if we can abstract it, that will be better